media/css/externalpages/pocket/components/pocket-features.scss

```diff
@@ -0,0 +1,237 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/split';
+@import '~@mozilla-protocol/core/protocol/css/base/elements/containers';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-family';
+
+@import '../utils/variables';
+@import '../includes/nav';
+@import '../includes/footer';
+
+.mzp-c-split {
+    padding: 64px 0;
+}
+
+.pocket-logo-inline {
+    @include image-replaced;
+    background: url('/media/img/externalpages/pocket/new-tab/pocket-logo-inline.svg') no-repeat;
+    display: inline-block;
+    height: 20px;
+    vertical-align: middle;
+    width: 20px;
+}
+
+.section-heading {
+    @include text-title-sm;
+    font-family: $font-sans-alt;
+    font-weight: 500;
+    margin-top: $spacing-sm;
+}
+
+.sub-section-heading {
+    @include text-title-xs;
+    font-family: $font-sans-alt;
+    font-weight: 500;
+}
+
+.tagline {
+    font-family: $font-sans;
+    @include text-body-sm;
+    text-transform: uppercase;
+}
+
+.section-lede {
+    @include text-body-xl;
+    font-family: $font-sans;
+}
+
+.app-store-badges {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    a {
+        width: 150px;
+        height: 100%;
+
+        &.apple-store {
+            margin-bottom: $spacing-md;
+        }
+
+        img {
+            max-height: 50px;
+            width: 100%;
+        }
+    }
+
+    @media #{$mq-md} {
+        justify-content: flex-start;
+        flex-direction: row;
+
+        .apple-store {
+            margin-bottom: 0 !important;
+            margin-right: $spacing-md;
+        }
+    }
+}
+
+.personal-library {
+    text-align: center;
+    margin: $layout-lg auto;
+
+    .app-store-badges {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        a {
+            width: 210px;
+
+            img {
+                max-height: 100%;
+            }
+        }
+    }
+}
+
+.pocket-premium {
+    border-top: 1px solid $color-divider;
+    padding: $spacing-sm;
+    transition: background-color 0.2s;
```

Comment:
**Question**: should the transition property have a third argument?

Reply:
Not always! In most cases, if timing is the same with hover-on and hover-off, then we just need 1 property to define the time for the transition and 1 other property for what attribute it should tackle